build-farm/platform-specific-configurations/linux.sh

```diff
@@ -31,10 +31,12 @@ fi
 if [ "$(pwd | wc -c)" -gt 83 ]; then
   # Use /tmp for alpine in preference to $HOME as Alpine fails gpg operation if PWD > 83 characters
   # Alpine also cannot create ~/.gpg-temp within a docker context
-  GNUPGHOME="$(mktemp --directory /tmp/.gpg-temp.XXXXX)"
+  GNUPGHOME="$(mktemp -d /tmp/.gpg-temp.XXXXX)"
```

Comment:
Feels like it's nicer to make it consistent with six `X` for both platforms:
```suggestion
  GNUPGHOME="$(mktemp -d /tmp/.gpg-temp.XXXXXX)"
```